vsts/pipelines/autoUpdater.yml

```diff
@@ -10,6 +10,11 @@ steps:
   inputs:
     version: 3.1.x
 
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+  displayName: 'Component Detection - OSS Compliance'
+  inputs:
+    ignoreDirectories: '$(Build.SourcesDirectory)/tests/SampleApps'
```

Comment:
@arroyc should we add `ignoreDirectories: '$(Build.SourcesDirectory)/tests/Oryx.Integration.Tests/bin/Debug/SampleApps'`? I see this path triggering alerts too: https://devdiv.visualstudio.com/DevDiv/_componentGovernance/107100/alert/3859190?typeId=8187690

Reply:
thanks you noticed this ... as discussed over teams channel .. I'm going to ignore the test directory itself